packages/voyager-keycloak/src/KeycloakSecurityService.ts

```diff
@@ -65,9 +69,11 @@ export class KeycloakSecurityService implements SecurityService {
       store: memoryStore
     }))
 
-    this.keycloak = new Keycloak({
-      store: memoryStore
-    }, this.keycloakConfig)
+    if (!this.keycloak) {
+      this.keycloak = new Keycloak({
+        store: memoryStore
```

Comment:
@darahayes Unrelated to this it using in-memory lib for session may be tech debt? 
Do you thin is worth issue?

Reply:
Absolutely worth it